pyvenv.cfg

```diff
@@ -0,0 +1,3 @@
+home = /usr/bin
+include-system-site-packages = false
+version = 3.6.9
```

Comment:
Please add this file to .gitignore or better yet, delete it and switch to using `venv` (it's also integrated in PyCharm).